app/src/main/java/com/example/bigowlapp/activity/SearchContactsToSupervise.java

```diff
@@ -107,8 +108,16 @@ public void onItemClick(AdapterView<?> arg0, View v, int position, long arg3) {
                             @Override
                             public void onComplete(@NonNull Task<QuerySnapshot> task) {
                                 if (task.isSuccessful()) {
-                                    if (!task.getResult().isEmpty())
+                                    if (!task.getResult().isEmpty()){
```

Comment:
Please use `CTRL-SHIFT-L`  to clean up the code in this file.

Reply:
changes have been made